src/languages/en.js

```diff
@@ -348,6 +352,11 @@ export default {
             uploadPhoto: 'Upload Photo',
             requestCall: 'Request a call',
         },
+        card: {
+            tagline: 'The Smartest corporate card in the room.',
+            publicCopy: "In order to use the Expensify Card you must use your company's private domain. Go ahead and add your private email address as a secondary login [here].",
```

Comment:
Guessing this is a `@todo` and we still need to do something about the `[here]`? Seems like maybe it's intended to be a link.

Reply:
https://github.com/Expensify/Expensify.cash/pull/3602#discussion_r652253805